.github/workflows/build.yml

```diff
@@ -24,6 +24,11 @@ jobs:
           node-version: "lts/*"
           cache: "npm"
 
+      - name: Test
```

Comment:
In the future I may break this out into it's own task/check, but for now keeping it part of build.